pkg/sink/sink.go

```diff
@@ -133,13 +133,14 @@ func (r Sink) processTrigger(t *triggersv1.EventListenerTrigger, request *http.R
 
 	rt, err := template.ResolveTrigger(*t,
 		r.TriggersClient.TektonV1alpha1().TriggerBindings(r.EventListenerNamespace).Get,
+		r.TriggersClient.TektonV1alpha1().ClusterTriggerBindings().Get,
 		r.TriggersClient.TektonV1alpha1().TriggerTemplates(r.EventListenerNamespace).Get)
 	if err != nil {
 		log.Error(err)
 		return err
 	}
 
-	params, err := template.ResolveParams(rt.TriggerBindings, finalPayload, header, rt.TriggerTemplate.Spec.Params)
+	params, err := template.ResolveParams(rt.TriggerBindings, rt.ClusterTriggerBindings, finalPayload, header, rt.TriggerTemplate.Spec.Params)
```

Comment:
It would be probably better if we pass `ResolvedTrigger` instead of passing individual members/submembers.